### ISSUE TITLE
JSON_data doesn't support $include in artists(), albums() like XML_data does

### ISSUE THREAD

**Issue by lachlan-00** [ampache/ampache#2350]
Just found a difference between XML and JSON @AshotN 

the functions aren't allowing the extra album/artist/song data that xml data.

```
            // Handle includes
            if (in_array("songs", $include)) {
                $songs = self::songs($album->get_songs(), array(), false);
            } else {
                $songs = $album->song_count;
            }
```

**Comment by AshotN** [comment]
I just looked through Git history. My JSON API pull request was made Jun 17, 2016. The $includes feature was added by @Phyks on July 25, 2016. Perhaps that's why I missed it.

Will fix!

**Comment by lachlan-00** [comment]
lol, i thought as much. i really like how it took almost 4 years for this to get into develop.

**Comment by AshotN** [comment]
Resolved by #2357
